src/eemo_log.c

```diff
@@ -167,7 +167,7 @@ void eemo_log(const int log_at_level, const char* file, const int line, const ch
 
 	if (log_at_level == EEMO_LOG_DEBUG)
 	{
-		static char debug_buf[8192];
+		static char debug_buf[8192 - 128];
 		vsnprintf(debug_buf, 8192, format, args);
```

Comment:
This seems like an odd change, since the call to `vsnprintf` assumes a buffer size of 8192 bytes? Looks like a buffer overrun waiting to happen?